### ISSUE TITLE
Video width 1072 instead of 1080

### ISSUE THREAD

**Issue by dx777** [NextLevel/NextLevel#239]
I did found a similar issue #178. But using the custom aspectRatio type not helping.
I tried different combinations, all together, or one by one, but the result is always the same.

This is what I tried:
```
videoConfiguration.preset = .hd1920x1080
videoConfiguration.aspectRatio = .custom(w: 9, h: 16)
videoConfiguration.dimensions = CGSize(width: 1080, height: 1920)
videoConfiguration.aspectRatio = .widescreenLandscape
// That's weird I have to use . widescreenLandscape, since it should be .widescreen for (9:16 ratio)

```

Other parameters that I have in my videoConfiguration:
```
videoConfiguration.maxKeyFrameInterval = 30
videoConfiguration.codec = .h264
videoConfiguration.bitRate = 7700000
```

**Comment by danieljvdm** [comment]
I'm having the same issue. Any of the following config settings results in a `1072x1920` video.

```
nextLevel.videoConfiguration.aspectRatio = .instagramStories
nextLevel.videoConfiguration.aspectRatio = .custom(w: 9, h: 16)
nextLevel.videoConfiguration.dimensions = CGSize(width: 1080, height: 1920)
```

**Comment by rojo69** [comment]
Fixed by PR #267
Assign nil or any suitable block size, even dividable by the desired video dimensions:
`nextLevel.videoConfiguration.sizeDivisibleBy = nil`
